### PR TITLE
fix(video): aligned to design & addressed youtube loading issue

### PR DESCRIPTION
**Pull request by sahmad-merative**
Multiple Youtube video issue fixes

## Issue

UI fixes : fonts, Spacing, and alignments of text and image fixes

- Add multiple YouTube videos on a page (I have added few videos on https://main--merative2--sahmad-merative.hlx.page/drafts/saad/leadspace-test).
- Play any one of the videos in a modal (popup).
- Close the modal before the video finishes playing.
- Open another video modal that contains a different YouTube video.
- Notice that the video from the previous modal resumes playing in the background from the point where the first modal was closed.

Fixes # [MERATIVE-784](https://jira.sdlc.merative.com/browse/MERATIVE-784)
[MERATIVE-796](https://jira.sdlc.merative.com/browse/MERATIVE-796)

## Description
aligned design according to figma
fixed multiple youtube video on a page issue

## Design Specs

- Figma Link -https://www.figma.com/file/vw24IPXXt4vCh9BNumka9A/Web--Merative-Digital-Design-System-2.0?type=design&node-id=1641%3A13366&mode=design&t=leAsxbNftxoq6gTd-1

## Test URLs
  
- Before (Changes from `main`): https://main--merative2--hlxsites.hlx.page/block-library/blocks/video
- After (Changes from this PR):https://784-video-promo--merative2--sahmad-merative.hlx.page/drafts/saad/leadspace-test
- https://784-video-promo--merative2--sahmad-merative.hlx.page/block-library/blocks/video
  
## Testing Instruction

> If applicable, please describe the tests that you ran to verify your changes. Provide instructions and link to the `hlx` deploy preview so that QA and the design team can provide proper testing.

image

image

image

image

### CODE REVIEW

**Comment by proeung** [comment]
@sahmad-merative Just a couple of notes before I start to review this PR.

- Please make sure you have the proper title for the PR. Currently, it's set to `784 video promo` and it needs to be descriptive and follows the conventional commit message format (eg. `fix(video): aligned to design & addressed youtube loading issue`
- Make sure the PR description area has a proper change list.
- Provide testing instruction for QA (@sharathmrft ) and stakeholders (@keith-kaplan) - See example here - https://github.com/hlxsites/merative2/pull/302#issue-1825111151

**Review by sharathmrft — CHANGES_REQUESTED** [review]
@sahmad-merative 
1.Video is  not playing for Carbon with video on left.

**Comment by sharathmrft** [comment]
@sahmad-merative 

Background You tube Video player issue is Tested it is working as expected

@proeung

**Review by sachinmesh — CHANGES_REQUESTED** [review]
1. Eyebrow font style is wrong.
2. Eyebrow - 32px spacing - Titile - 32px spacing - Bodycopy
3. The component should be end to end , no padding on left and right side.
4. Width of this video block should be 744 x 420px.


     
image


@sahmad-merative @sharathmrft @proeung

File: blocks/video/video.js [hunk 1]
```diff
@@ -104,42 +97,7 @@ const decorateVideoLink = (link, label = 'Play') => {
...
-  const onPlayerReady = (event) => {
-    playerMap[videoId] = event.target;
-  };
-
-  // Create a new iframe element
-  const iframe = document.createElement('iframe');
-  iframe.src = `https://www.youtube.com/embed/${videoId}`;
-
-  // Add the data-hj-allow-iframe attribute
-  iframe.setAttribute('data-hj-allow-iframe', '');
```
Comment (proeung) [inline blocks/video/video.js:122]: This code should not be removed. We need to add the `data-hj-allow-iframe` to the YT iframe.

**Review by proeung — CHANGES_REQUESTED** [review]
@sahmad-merative Looks good! I just have a comment about the removal of the `data-hj-allow-iframe` attribute that's needed within the YT iframe code.

Also, @amol-anand or @helms-charity Can you help review this PR? There are a good amount of code changes within the `video.js` that would be great for you to take a look and make sure we're not missing anything.

**Review by proeung — APPROVED** [review]
@sahmad-merative Thanks for addressing my feedback. Everything looks good from a code perspective.

I'll defer to @sachinmesh and @sharathmrft to verify the design QA and browser testing portions.

**Review by sharathmrft — APPROVED** [review]
@sahmad-merative You tube player is verified it is working as expected and verified in different browsers as well.
@proeung @Shalini-SB

**Review by sachinmesh — APPROVED** [review]

**Pull request merged**